### PR TITLE
fix: 修复趋势分析表导出问题

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 📝 Description
趋势分析表即使有多个层级的行头，`rows`配置也只有`extra`，此时，角头部分需要增加占位来对齐行头的单元格，以下图为例：
image

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| image | image     |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/utils/export/index.ts [hunk 1]
```diff
@@ -271,9 +272,9 @@ export const copyData = (
   );
 
   // get max query property length
-  const rowLength = rowLeafNodes.reduce((pre, cur) => {
-    const length = cur.query ? Object.keys(cur.query).length : 0;
```
Comment (wjgogogo) [inline packages/s2-core/src/utils/export/index.ts:275]: 趋势表通过query判断层级并不对，指标B位于第二层，但是query只有一个字段：
image
所以改用node层级做判断

**Review by lijinke666 — DISMISSED** [review]

**Review by lcx-seima — APPROVED** [review]
LG

**Pull request merged**